app/views/edit/deployment-config.html

```diff
@@ -0,0 +1,373 @@
+<default-header class="top-header"></default-header>
+<div class="wrap no-sidebar">
+  <div class="sidebar-left collapse navbar-collapse navbar-collapse-2">
+    <navbar-utility-mobile></navbar-utility-mobile>
+  </div>
+  <div class="middle surface-shaded">
+    <!-- Middle section -->
+    <div class="middle-section" ng-show="deploymentConfig">
+      <div class="middle-container">
+        <div class="middle-content">
+          <div class="container surface-shaded">
+            <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+            <alerts alerts="alerts"></alerts>
+            <div ng-if="!loaded">Loading...</div>
+            <div ng-if="loaded">
+              <h1>
+                Edit Deployment Config {{deploymentConfig.metadata.name}}
+                <small>&mdash; {{strategyData.type | startCase}} Deployment Strategy</small>
+              </h1>
+              <fieldset ng-disabled="disableInputs">
+                <form class="edit-form" name="form" novalidate ng-submit="save()">
+                  <div class="row">
+                    <div class="col-lg-12">
+                      <div class="section">
+                        <h3>Deployment Strategy</h3>
+                        <dl class="dl-horizontal left">
+                          {{strategyParamsName}}
+                          <div class="form-group strategy-name">
+                            <label class="picker-label">Strategy Type</label>
+                            <ui-select ng-model="strategyData.type" search-enabled="false" ng-change="strategyChanged()">
+                              <ui-select-match>{{$select.selected}}</ui-select-match>
+                              <ui-select-choices repeat="strategyType in deploymentConfigStrategyTypes">
+                                {{strategyType}}
+                              </ui-select-choices>
+                            </ui-select>
+                            <div>
+                              <span ng-switch="strategyData.type">
+                                <span class="help-block" ng-switch-when="Recreate">
+                                  Recreate strategy has basic rollout behavior and supports lifecycle hooks for injecting code into the deployment process.
```

Comment:
I'd start each of these strategy descriptiong with "The".

"The recreate strategy has basic..."